iot/mqtt_example/cloudiot_mqtt_example_nodejs.js

```diff
@@ -122,18 +137,42 @@ function createJwt (projectId, privateKeyFile, algorithm) {
 // messageCount.
 // [START iot_mqtt_publish]
 function publishAsync (messageCount, numMessages) {
```

Comment:
`messageCount` and `numMessages` are basically synonymous, which one is used for what? Otherwise LGTM

Reply:
Done